devtools/python-output-table-conversion-debug.ipynb

```diff
@@ -66,53 +66,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "id": "48c79723-28b5-4091-9590-7dedfd23d7df",
    "metadata": {
     "tags": []
    },
-   "outputs": [
```

Comment:
We should also update the last step in `devtools/python-output-table-conversion-debug.ipynb`to to reflect the new way to update the PudlTabl methods.

Reply:
Updated